src/ResourceManager/Common/Commands.Common.Strategies/IAsyncCmdlet.cs

```diff
@@ -13,18 +13,23 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Commands.Common.Strategies;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Microsoft.Azure.Commands.Compute.Strategies
+namespace Microsoft.Azure.Commands.Common.Strategies
 {
-    interface IAsyncCmdlet
+    public interface IAsyncCmdlet
```

Comment:
Please provide documentation comments for this public interface and each of its methods and properties.